ci/tests/rp-flux/test.sh

```diff
@@ -5,7 +5,7 @@ cp resource_flux.json ~/.radical/pilot/configs/resource_flux.json
 
 cd radical.pilot
 echo '--- smoke test'
-./examples/00_getting_started.py flux.localhost_flux
+./examples/00_getting_started.py 'local.localhost_flux'
```

Comment:
should be `flux`, since the file name is `resource_flux`
(btw I've prepared PR #135 to have RP tests updated)

Reply:
Fixed.